x-pack/plugins/index_management/public/application/components/index_templates/legacy_index_template_deprecation.tsx

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { i18n } from '@kbn/i18n';
+import { EuiCallOut, EuiLink } from '@elastic/eui';
+import { ScopedHistory } from 'kibana/public';
+import { reactRouterNavigate } from '../../../shared_imports';
+import { documentationService } from '../../services/documentation';
+
+interface Props {
+  history?: ScopedHistory;
+  showCta?: boolean;
+}
+
+export const LegacyIndexTemplatesDeprecation: React.FunctionComponent<Props> = ({
+  history,
+  showCta,
+}) => {
+  return (
+    <EuiCallOut
+      title={i18n.translate('xpack.idxMgmt.legacyIndexTemplatesDeprecation.title', {
+        defaultMessage:
+          'Legacy index templates are deprecated in favor of composable index templates',
```

Comment:
Not blocking, but this title is a bit long. I'd consider moving the composable index templates suggestion to the message text.

<img width="447" alt="Screen Shot 2021-06-14 at 2 12 31 PM" src="https://user-images.githubusercontent.com/40268737/121939154-ac63d680-cd1a-11eb-8bc9-85e484ce9672.PNG">


Feel free to ignore this suggestion if wanted.